pkg/webhook/stats_reporter.go

```diff
@@ -85,6 +105,20 @@ func register() error {
 			Aggregation: view.Distribution(0.001, 0.002, 0.003, 0.004, 0.005, 0.006, 0.007, 0.008, 0.009, 0.01, 0.02, 0.03, 0.04, 0.05),
 			TagKeys:     []tag.Key{admissionStatusKey},
 		},
+		{
+			Name:        mutationRequestCountMetricName,
+			Description: "The number of requests that are routed to mutation webhook",
+			Measure:     mutationResponseTimeInSecM,
+			Aggregation: view.Count(),
+			TagKeys:     []tag.Key{mutationStatusKey},
+		},
+		{
+			Name:        mutationRequestDurationMetricName,
+			Description: mutationResponseTimeInSecM.Description(),
+			Measure:     mutationResponseTimeInSecM,
+			Aggregation: view.Distribution(0.001, 0.002, 0.003, 0.004, 0.005, 0.006, 0.007, 0.008, 0.009, 0.01, 0.02, 0.03, 0.04, 0.05),
```

Comment:
do we know if this distribution works? should we add a todo to adjust later?

Reply:
Done
TODO added

Reply:
We should absolutley adjust to have a maximum of 3 seconds (default timeout). There is an issue for the validation metrics where we say we should do this.

Reply:
Can we do this in another PR? This sounds like a different issue than adding new reporting for mutation. There we can change it for both validation and mutation.

Reply:
+1